Make one-compiler an optional dependency (#198)
Samsung/TICO: README.md

0. Prerequisites

- Python 3.10
- (Optional) [one-compiler 1.30.0](https://github.com/Samsung/ONE/releases/tag/1.30.0)

>   - It's only required if you want to run inference with the converted Circle model. If you're
>    only converting models and don't plan to run them, this dependency is not needed.

Reviewer: ```suggestion
  - It is only required if you intend to run inference with the converted Circle model. If you are only converting models without running them, this dependency is not needed.
```
